src/components/TodoLayout/TodoCreate/TodoCreate.tsx

```diff
@@ -49,11 +49,11 @@ export const CreateTodo = () => {
   };
 
   const handleChangeInput = (e: ChangeEvent<HTMLInputElement>) => {
-    if (e.target.value.trim().length >= MAX_TITLE_LENGTH) {
-      dispatch(setNotification({ title: TodoErrorMessages.MAX_LENGTH, type: NotificationType.ERROR }));
+    if (e.target.value.trim().length >= TodoValidateData.MAX_TITLE_LENGTH) {
+      dispatch(setNotification({ title: TodoNotificationMessages.MAX_LENGTH, type: NotificationType.ERROR }));
       return setTitleStoreValue(todoTitle.replace(e.target.value, ""));
     }
-    setTitleStoreValue(checkOnValidField(e.target.value));
+    setTitleStoreValue(isValidField(e.target.value));
```

Comment:
You use e.target.value a few times in this function, why don't you create a variable?